### ISSUE TITLE
plural "other" as fallback and `useFallbackTranslations` priority problem

### ISSUE THREAD

**Issue by kyle-seongwoo-jun** [aissat/easy_localization#400]
I tested https://github.com/aissat/easy_localization/pull/389, but I think it has a single problem.

#### en.json
```json
"seconds": {
  "one": "{} second",
  "other": "{} seconds"
}
```

#### ko.json
```json
"seconds": {
  "other": "{}초"
}
```

#### in code
``` dart
// in main
EasyLocalization(
  fallbackLocale: Locale('en'),
  useFallbackTranslations: true
  ...
);

'seconds'.plural(1)
// en: 1 second
// (expected) ko: 1초
// (actaully) ko: 1 second
```

When `useFallbackTranslations` is true, `fallbackLocale` is used as fallback, not "other".

I think "other" on the same locale should have more priority than the same key on `fallbackLocale`.

Thank you.

**Comment by Overman775** [comment]
@aissat please recheck new plural algoritm

**Comment by ValentinVignal** [comment]
Hello, is there any progress on this issue?

**Comment by kyle-seongwoo-jun** [comment]
It wasn't fixed on #440 and #470
